src/index.js

```diff
@@ -1,13 +1,13 @@
 import React, { useState, useEffect } from 'react';
-import ReactDOM from 'react-dom';
-import qs from 'query-string';
+import * as ReactDOM from 'react-dom';
+import * as qs from 'query-string';
```

Comment:
I don't see `qs` being used too.

Reply:
it's used here `const returnedId = qs.parse(window.location.search)['id'];` (line number 50)